x-pack/plugins/cases/public/components/user_actions/comment/alert_event.tsx

```diff
@@ -42,7 +42,7 @@ const AlertCommentEventComponent: React.FC<Props> = ({
     },
     [ruleId, onRuleDetailsClick]
   );
-  const detectionsRuleDetailsHref = getRuleDetailsHref(ruleId);
+  const detectionsRuleDetailsHref = getRuleDetailsHref?.(ruleId) ?? '';
```

Comment:
I understand this component's purpose is to show a link but all the possible parameter to display the link are optional. Won't this potentially end up displaying empty links? 

Maybe we need an empty-state handling for this component?